spdx-utils/src/main/resources/licenserefs/LicenseRef-ort-google-maps-tos

```diff
@@ -0,0 +1,288 @@
+Google Maps Platform Terms of Service
```

Comment:
Please note that the contents of the sub-directories below the ` spdx-utils/src/main/resources` directory are auto-generated by Gradle tasks in the `spdx-utils` module. As such, we must not manually add files here. And we actually never had the need, until now it seems. I'm not yet sure how to deal with this. The easiest for us would be to simply ask ScanCode to start detecting this license. But then again this is not really a license as it would appear in a `LICENSE` file, but rather terms of use. So maybe we need to extend our mechanism to also allow manually added files like this.

Any opinions?

Reply:
IIRC ScanCode is not limited to just licenses but also has terms of use like things. E.g. the crypto export limitations maybe?!
This was also the first thing popping up to me when seeing this, e.g. why not add it in ScanCode?!

Reply:
From my understanding adding it to ScanCode would require that there is some code where this license could be detected.
As far as I know the library where this was found in the metadata is not published as open source.
Would it still be feasible to add this license to ScanCode?

Reply:
I believe so, yes. At least with its `-p` option ScanCode has some understanding of package metadata, and it could emit this finding when the string "Google Maps Platform Terms of Service" is found in the `<license>` tag of a POM, even if no `LICENSE` file with the full text would be scanned.

But I would argue that ScanCode should *also* add support for detecting this license from the license text, for completeness.

@MarcelBochtler could you please open an issue at the upstream ScanCode project for this, to see what @pombredanne thinks about this?

Reply:
Created an issue for ScanCode to add the `Google Maps Platform Terms of Service`.

Reply:
Link? 😉

Reply:
[Link to ScanCode issue](https://github.com/nexB/scancode-toolkit/issues/2322)